.s2i/bin/assemble

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+/usr/libexec/s2i/assemble
+npm config set proxy $http_proxy
```

Comment:
Should you also set the https proxy? I assume it's not a problem right now, or is made not a problem by setting the registry explicitly to an http registry, but it can't imagine it causing more problems than it could potentially solve.